bonus/bitcoin/ordisrespector.md

```diff
@@ -312,23 +262,110 @@ libtoolize: copying file 'build-aux/ltmain.sh'
 [...]
 ```
 
-* The next command will configure the Berkeley DB path, set the next environment variable
+* The next command will download and build all Bitcoin Core dependencies
 
 ```bash
-export BDB_PREFIX="/tmp/bitcoin-$VERSION/db4"
+make -C depends -j$(nproc) NO_QR=1 NO_QT=1 NO_NATPMP=1 NO_UPNP=1 NO_USDT=1
+```
+
+Expected output:
+
+```
+make: Entering directory '/tmp/bitcoin-28.1/depends'
+Fetching boost_1_81_0.tar.gz from https://archives.boost.io/release/1.81.0/source/
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  133M  100  133M    0     0  39.0M      0  0:00:03  0:00:03 --:--:-- 39.0M
+/tmp/bitcoin-28.1/depends/work/download/boost-1.81.0/boost_1_81_0.tar.gz.temp: OK
+Extracting boost...
+/tmp/bitcoin-28.1/depends/sources/boost_1_81_0.tar.gz: OK
+Preprocessing boost...
+Configuring boost...
+Building boost...
+Staging boost...
+Postprocessing boost...
+Caching boost...
+Fetching libevent-2.1.12-stable.tar.gz from https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
+100 1075k  100 1075k    0     0  1117k      0 --:--:-- --:--:-- --:--:-- 3812k
+/tmp/bitcoin-28.1/depends/work/download/libevent-2.1.12-stable/libevent-2.1.12-stable.tar.gz.temp: OK
+Extracting libevent...
+/tmp/bitcoin-28.1/depends/sources/libevent-2.1.12-stable.tar.gz: OK
+Preprocessing libevent...
+patching file CMakeLists.txt
+patching file cmake/AddEventLibrary.cmake
+patching file CMakeLists.txt
+Configuring libevent...
+-- The C compiler identification is GNU 11.4.0
+-- Detecting C compiler ABI info
+-- Detecting C compiler ABI info - done
+-- Check for working C compiler: /usr/bin/gcc - skipped
+-- Detecting C compile features
+-- Detecting C compile features - done
+-- Found Git: /usr/bin/git (found version "2.34.1") 
+fatal: not a git repository (or any of the parent directories): .git
+-- Performing Test check_c_compiler_flag__Wall
+-- Performing Test check_c_compiler_flag__Wall - Success
+-- Performing Test check_c_compiler_flag__Wextra
+-- Performing Test check_c_compiler_flag__Wextra - Success
+[...]
 ```
 
 * Pre-configure the installation, we will discard some features and include others. Enter the complete next command in the terminal and press enter
 
 ```sh
 ./configure \
-   BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" BDB_CFLAGS="-I${BDB_PREFIX}/include" \
+  CONFIG_SITE=depends/x86_64-pc-linux-gnu/share/config.site \
```

Comment:
Note: this path (x86_64-pc-linux-gnu) only works verbatim on amd64 machines, but I believe Minibolt assumes the user has an x86-64 machine.

If not, it could be changed for something more dynamic that works on any architecture: https://github.com/BcnBitcoinOnly/docker-knots/blob/master/Dockerfile#L53

Reply:
It is good; I may need to modify that for RaMiX, but I'll take care of that myself.